src/cljdoc/util/fixref.clj

```diff
@@ -57,25 +56,42 @@
       (str from-uri-map anchor)
       (str scm-base root-relative))))
 
-(defn fix-image
-  [file-path src {:keys [scm-base]}]
-  (let [suffix (when (.endsWith src ".svg") "?sanitize=true")]
+(defn- scm-rev [scm]
+  (or (:name (:tag scm))
+      (:commit scm)))
+
+(defn- local-raw-server
+  "This is overly inflexible for the first version.
+  When testing locally we point to simple static http server."
+  [scm]
+  (when (scm/fs-uri (:url scm)) "http://localhost:9090/"))
+
+(defn- raw-base-url [scm]
+  (or (local-raw-server scm)
+      (str (:url scm)
+           "/raw/"
+           (scm-rev scm) "/")))
```

Comment:
Reviewing this code I was a little concerned that `raw-base-url` is different than what has previously been done here
<img width="472" alt="CleanShot 2020-06-11 at 22 40 30@2x" src="https://user-images.githubusercontent.com/97496/84436968-97022b00-ac34-11ea-9062-f4b7f5246143.png">

It seems that instead of using `https://raw.githubusercontent.com/` we now use the `/raw/` URL segment to access raw versions of the files. If these can be used interchangeably then that change is fine with me, just trying to understand if/why something changed here. 🙂

Reply:
Thanks for the concern, this is certainly a rusty PR and merits review!  This change is described in the PR message under "1) raw url now more generic".

Reply:
Duh, should have referred back to the PR description before asking haha 😄 Sounds good then!